scripts/start.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+# used for loading correct manifest
+export COMMIT_HASH=`cat COMMIT_HASH.txt`
```

Comment:
Where is this used? I can't find a usage of it.

Reply:
So the commit has can be used via the `bucket-assets` module to get the manifest from our public S3 bucket - see https://github.com/artsy/post_mortems/issues/35#issuecomment-409166093